drake/bindings/matlab/+autodiffutils/toAutoDiff.m

```diff
@@ -0,0 +1,18 @@
+function obj = toAutoDiff(value, derivatives)
+
+if nargin < 2
+  derivatives = eye(numel(value));
+end
+
+if size(value, 2) == 1
+  if size(value, 1) <= 73
+    obj = autodiffutils.AutoDiffVectorMax73(value, derivatives);
```

Comment:
The matlab files that Swig generates get put in a `+autodiffutils` package inside pod-build, which is added to the Matlab path. It turns out that you can have multiple folders with the same package name in Matlab, and all of them will get searched for the appropriate methods, so this file looks to the users like its in the same package as the files it references.